test/maps/market_trading.wmf/scripting/tribes/custom_building/init.lua

```diff
@@ -0,0 +1,63 @@
+dirname = "test/maps/market_trading.wmf/" .. path.dirname(__file__)
+
+tribes:new_productionsite_type {
+   msgctxt = "barbarians_building",
+   name = "barbarians_custom_building",
+   -- TRANSLATORS: This is a building name used in lists of buildings
+   descname = pgettext("custom_building", "Custom Building"),
+   helptext_script = dirname .. "helptexts.lua",
+   icon = dirname .. "menu.png",
+   size = "small",
+
+   buildcost = {
+      log = 3
+   },
+   return_on_dismantle = {
+      log = 2
+   },
+
+   animations = {
+      idle = {
+         pictures = path.list_files(dirname .. "idle_??.png"),
+         hotspot = { 43, 45 },
+      },
+      build = {
+         pictures = path.list_files(dirname .. "build_??.png"),
+         hotspot = { 43, 45 },
+      },
+      unoccupied = {
+         pictures = path.list_files(dirname .. "unoccupied_??.png"),
+         hotspot = { 43, 45 },
+      },
+   },
+
+   aihints = {
+      basic_amount = 1,
+   },
+
+   working_positions = {
+      barbarians_custom_worker = 1
+   },
+
+   outputs = {
```

Comment:
This one can also go now. And convert the animations for *all* custom units to the new format please.